lib/netext/grpcext/conn.go

```diff
@@ -196,25 +197,28 @@ func (statsHandler) TagRPC(ctx context.Context, _ *grpcstats.RPCTagInfo) context
 func (h statsHandler) HandleRPC(ctx context.Context, stat grpcstats.RPCStats) {
 	state := h.vu.State()
 	tags := getTags(ctx)
+	if tags == nil {
+		// FIXME: this is an RPC for reflection
+		// find how to handle it properly
```

Comment:
as far as I remember - we just emitted with empty tags for the reflect https://github.com/grafana/k6/pull/2160#pullrequestreview-784595048 (and all the other discussion in that PR)

Reply:
https://github.com/grafana/k6/pull/2634

Reply:
#2634 has been merged so this has been resolved.